### ISSUE TITLE
Bug in image rendering of dative bonds

### ISSUE THREAD

**Issue by IvanChernyshov** [rdkit/rdkit#3056]
In some cases, dative bonds are painted in the wrong color of the end atom:
```
Chem.MolFromSmiles('N->1CCN->[Cu++]1')
```
![image](https://user-images.githubusercontent.com/17598902/78022439-9825fa80-735d-11ea-82de-d9dbb31e1698.png)

or are even invisible, while the molecular graph is correct:
```
mol = Chem.MolFromSmiles('[Cu++]->1->2.N1CCN2')
mol
```
￼![image](https://user-images.githubusercontent.com/17598902/78022574-d91e0f00-735d-11ea-8c6e-124ec1700118.png)
```
mol.Debug()
RDKit INFO: Atoms:
RDKit INFO: 	0 29 Cu chg: 2  deg: 2 exp: 0 imp: 0 hyb: 6 arom?: 0 chi: 0
RDKit INFO: 	1 7 N chg: 0  deg: 2 exp: 2 imp: 1 hyb: 4 arom?: 0 chi: 0
RDKit INFO: 	2 6 C chg: 0  deg: 2 exp: 2 imp: 2 hyb: 4 arom?: 0 chi: 0
RDKit INFO: 	3 6 C chg: 0  deg: 2 exp: 2 imp: 2 hyb: 4 arom?: 0 chi: 0
RDKit INFO: 	4 7 N chg: 0  deg: 2 exp: 2 imp: 1 hyb: 4 arom?: 0 chi: 0
RDKit INFO: Bonds:
RDKit INFO: 	0 1->2 order: 1 conj?: 0 aromatic?: 0
RDKit INFO: 	1 2->3 order: 1 conj?: 0 aromatic?: 0
RDKit INFO: 	2 3->4 order: 1 conj?: 0 aromatic?: 0
RDKit INFO: 	3 0->1 order: 17 conj?: 0 aromatic?: 0
RDKit INFO: 	4 0->4 order: 17 conj?: 0 aromatic?: 0
```

Unfortunately, I was not able to find any specific pattern of this bug.

**Comment by DavidACosgrove** [comment]
I'll have a look at this.  It was my fault, more than likely.

**Comment by DavidACosgrove** [comment]
Ivan, can you say which version of RDKIt you're using, please?  This is something I thought I'd fixed in 2020.03.1 (at least, the missing bonds bit) and I'm not reproducing the problem in the new version.

**Comment by greglandrum** [comment]
Closed by #3066
